packages/amplication-client/src/VersionControl/Build.tsx

```diff
@@ -244,17 +245,22 @@ const BuildHeader = ({ build, deployments }: BuildHeaderProps) => {
 
   const deployedClassName = `${CLASS_NAME}__header--deployed`;
 
+  const isDeployed =
+    deployments &&
+    deployments.length &&
+    deployments[0].status === models.EnumDeploymentStatus.Completed;
```

Comment:
can be greatly simplified with lodash head function

Reply:
If we use head we should still check that the results of head are not undefined before we check the status so I don't think it will greatly simplify...

const deployment = head(deployments);
const isDeployed = deployment && deployment.status === models.EnumDeploymentStatus.Completed;

Am I missing anything?